connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -232,8 +252,7 @@ private void initPluginLoader(String path) {
             if (CLASSPATH_NAME.equals(path)) {
                 scanUrlsAndAddPlugins(
                         getParent(),
-                        ClasspathHelper.forJavaClassPath().toArray(new URL[0]),
-                        null
```

Comment:
Thanks, the "unused parameter" warning from my IDE has been bugging me here about this for ages 🙏